src/App.tsx

```diff
@@ -1,10 +1,58 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Error } from './components/Error';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoItem } from './components/TodoItem';
+import { getTodos } from './api/todos';
+import { Todo, FilterStatus } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 11220;
 
 export const App: React.FC = () => {
+  const [selectedStatus, setSelectedStatus] = useState(FilterStatus.All);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [todosContent, setTodosContent] = useState<Todo[]>([]);
+
+  const todos = useMemo(() => {
```

Comment:
Consider naming your variables todos and filteredTodos

Reply:
agree with [comment](https://github.com/mate-academy/react_todo-app-loading-todos/pull/674/files#r1276541678)